python/pyspark/sql/functions.py

```diff
@@ -2038,13 +2038,22 @@ def _wrap_function(sc, func, returnType):
                                   sc.pythonVer, broadcast_vars, sc._javaAccumulator)
 
 
+class PythonUdfType(object):
+    # row-based UDFs
```

Comment:
Nit: Please update all `row-based UDFs` to `row-at-a-time UDFs`

Reply:
Sure, I'll update it.